worker-store-fs/src/test/java/com/hpe/caf/worker/datastore/fs/FileSystemDataStoreTest.java

```diff
@@ -17,10 +17,10 @@
 
 
 import com.hpe.caf.api.ConfigurationException;
-import com.hpe.caf.api.worker.DataStore;
-import com.hpe.caf.api.worker.DataStoreException;
-import com.hpe.caf.api.worker.FilePathProvider;
-import com.hpe.caf.api.worker.ReferenceNotFoundException;
+import com.hpe.caf.api.worker.*;
```

Comment:
You should try to avoid the star import.  It is considered bad practice:
http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport